src/pages/Search/SearchFilters.tsx

```diff
@@ -25,6 +25,7 @@ type SearchMenuFilterItem = {
     route: Route;
 };
 
+// Because we will add have AdvancedFilters, in future rename this component to `SearchTypeMenu|Tabs|Filters` to avoid confusion
```

Comment:
We can already rename this to `SearchStatusBar`

Reply:
Sorry but this can't be `SearchStatusBar` because this is the component that is currently (and for now still will be) displayed in LHN. So this is not the moment to rename it or it will confuse developers.

We should rename it I think when we are actually removing it from LHN - which this PR is not doing.
CC @adamgrzybowski @WojtekBoman